ytfzf

```diff
@@ -30,29 +30,39 @@ thumb_dir="$YTFZF_CACHE"/thumb
 [ -d $thumb_dir ] || mkdir -p $thumb_dir
 
 
+# DEP CHECK
+dep_ck () {
+	for dep in "$@"; do
+		command -v "$dep" 1>/dev/null || { printf "$dep not found. Please install it.\n" ; exit 2; }
+	done
+}
+dep_ck "jq" "youtube-dl" "$YTFZF_PLAYER";
```

Comment:
#51 `YTFZF_PLAYER` might be not a single command, example `devour mpv` or `mpv --some-option`

Reply:
Yeah, sorry. Didn't think about that. What I meant for was that by setting `mpv` in the dependency, one cannot use something like VLC on a system without mpv, without changing it in multiple places (sorry if that doesn't make sense) to avoid dependency errors.

Reverting the change to `dep_ck`.